api/views.py

```diff
@@ -62,14 +69,25 @@ def contact():
     return render_template('contact.html', page_title='Contact Us')
 
 
+# Renders insert internship page:
+@views.route('/insert-internship', methods=['GET'])
+def insert_internship():
+    """Insert internship page for users.
+    This function runs whenver the insert-internship page ('/insert-internship') is requested.
+        ex) -> localhost:5000/insert-internship
+    Returns the login page: which renders insert.html (insert internship page)
+    """
+    return render_template('insert.html', page_title='Insert Internship')
+
+
 # TODO: Find a better way to seed default data into database.
 @views.route('/temp', methods=['GET'])
 def temp():
-    from api import db, UserModel
-    me = UserModel(username='jventura3', 
-                   email='jventura3@gulls.salisbury.edu',
-                   password='a900a10c4d321f8338325b3cea14cbaa53a87bd3462bac578ab3966ea7ab9db8')
+    from api import db, UsersModel
+    me = UsersModel(username='jventura3',
+                    email='jventura3@gulls.salisbury.edu',
+                    password='a900a10c4d321f8338325b3cea14cbaa53a87bd3462bac578ab3966ea7ab9db8')
```

Comment:
Probably shouldn't commit this to the repository. Better to just put this creation code in a script somewhere (that gets ignored) so you can run it manually to add the user.